app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java

```diff
@@ -266,8 +266,8 @@ public void onClick(DialogInterface dialogInterface, int i) {
         // Create Components
         // TODO: Change type for drop downs once we have the right XML and Java for it.
         CheckBox check_DidLeave = postMatchBinding.checkboxDidLeave;
-        EditText drop_ClimbPosition = postMatchBinding.dropClimbingPosition;
-        EditText drop_Trap = postMatchBinding.dropTrap;
+//        EditText drop_ClimbPosition = postMatchBinding.dropClimbingPosition;
```

Comment:
you should remove these lines

Reply:
These the problem was that it wasn't added to XML yet so it errored, so we will want it later.